less/forum.less

```diff
@@ -5,10 +5,15 @@
  * LICENSE file that was distributed with this source code.
  */
 
-.PostMention, .UserMention {
-  .spoiler & {
-    background: #444;
-    color: transparent;
+span.spoiler {
+  background: #444;
+  color: transparent;
+  a, a:link, a:visited, a:hover, a:active {
     padding: 0;
+    color: transparent;
+    cursor: default;
+  }
```

Comment:
Could we turn the styling for links into this?

~~~css
* {
  color: transparent;
  background: transparent;
  pointer-events: none;
}
~~~

Would that still work? That would be super generic. :grin:

Reply:
It's also curious that the old code had an explicit background for the link (line 10), but your new proposal does not.